src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientBatchOpsImpl.java

```diff
@@ -380,6 +382,12 @@ public void partitionExecute(ObTableOperationResult[] results,
                                 "tablename:{} partition id:{} batch ops retry while meet ObTableMasterChangeException, errorCode: {} , retry times {}",
                                 tableName, partId, ((ObTableException) ex).getErrorCode(),
                                 tryTimes, ex);
+                        if (ex instanceof ObTableRoutingWrongException) {
+                            System.out.println("need fetchAll refresh");
+                            obTableClient.getOrRefreshTableEntry(tableName, true, obTableClient.isTableEntryRefreshIntervalWait(), true);
```

Comment:
getTable will call getOrRefreshTableEntry once, which means getOrRefreshTableEntry will be called twice? Can we merge into one getOrRefreshTableEntry?

Reply:
Fixed